sdk/typescript/src/transactions/Transaction.ts

```diff
@@ -99,19 +99,24 @@ export function isTransaction(obj: unknown): obj is Transaction {
 
 export type TransactionObjectInput = string | CallArg | TransactionObjectArgument;
 
-const modulePluginRegistry = {
-	buildPlugins: [] as TransactionPlugin[],
-	serializationPlugins: [] as TransactionPlugin[],
+interface TransactionPluginRegistry {
+	// eslint-disable-next-line @typescript-eslint/ban-types
+	buildPlugins: Map<string | Function, TransactionPlugin>;
+	// eslint-disable-next-line @typescript-eslint/ban-types
+	serializationPlugins: Map<string | Function, TransactionPlugin>;
+}
+
+const modulePluginRegistry: TransactionPluginRegistry = {
+	// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Do you still need this eslint disable here?

Reply:
nope